api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -138,6 +138,10 @@ message ExtAuthz {
   //         stat_prefix: blocker # This emits ext_authz.blocker.ok, ext_authz.blocker.denied, etc.
   //
   string stat_prefix = 13;
+
+  // Optional labels that will be passed to :ref:`labels<envoy_v3_api_field_service.auth.v3.AttributeContext.Peer.labels>` in
+  // :ref:`destination<envoy_v3_api_field_service.auth.v3.AttributeContext.destination>` if provided.
+  map<string, string> destination_labels = 15;
```

Comment:
do you expect these fields to be set at per route config so ext_authz get different labels per route? I'm thinking if we should use cluster metadata or something else instead of directly pass this in the config.

Reply:
Right now there is no such use case. But we never know.  Using cluster metadata seems an interesting idea. What is the advantage of using cluster metadata than this? Is n't cluster metadata need to be configured for every service if the choice of using ext authz is done by few services using envoy filter or request authentication (in Istio world)? Are there any downsides of specifying in config?

Reply:
Just thinking loud, I was assuming this labels would be overlapping with cluster metadata so it blows the config size in LDS/RDS. I'd rather put them in a generic place but ext_authz specific.

Reply:
Another idea I was thinking was since these are destination labels - Could we use bootstrap metadata some how? If labels are there in bootstrap metadata - we just pass it - May be we will have to agree on some standard key?

Reply:
> Another idea I was thinking was since these are destination labels - Could we use bootstrap metadata some how? If labels are there in bootstrap metadata - we just pass it - May be we will have to agree on some standard key?

Yeah that's my confusing point as well, if bootstrap metadata works why not use them? We can agree on standard key, or make the key configurable here.

Reply:
Ok. I think that makes sense to me. Should we start with agree on a key for now and if we need add config in future here?

Reply:
Should we standardize on `LABELS`  as the key to start with?

Reply:
@lizan What do you think about adding a config "bootstrap_metadata_labels_key" and pass labels from that key?